doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -480,9 +452,8 @@ automatically created.
       scrutinee’s head, recursively, casts being ignored. :cmd:`Hint Mode` is
       especially useful for typeclasses, when one does not want to support default
       instances and avoid ambiguity in general. Setting a parameter of a class as an
-      input forces proof-search to be driven by that index of the class, with ``!``
-      giving more flexibility by allowing existentials to still appear deeper in the
-      index but not at its head.
+      input forces proof search to be driven by that index of the class, with ``!``
+      allowing existentials to appear in the index but not at its head.
 
    .. note::
```

Comment:
The first note is specific to `Hint Extern` so should be moved in the `Hint Extern` description.